src/cunumeric/random/bitgenerator_curand.inl

```diff
@@ -0,0 +1,343 @@
+/* Copyright 2021-2022 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include <unistd.h>
+#include <sys/syscall.h>
+#include <sys/types.h>
+#include <mutex>
+
+#if 0  // for debugging
+#define VERBOSE_DEBUGGING
+#endif
+
+#ifdef VERBOSE_DEBUGGING
+
+static void printtid(int op)
+{
+  pid_t tid;
+  tid = syscall(SYS_gettid);
```

Comment:
Should this also be updated for consistence ? https://github.com/nv-legate/cunumeric/blob/9110ca5dfeea1ca4880554c0ed5b5fb150016a02/src/cunumeric/cuda_help.h#L119

Reply:
Yes, I'll update those helpers, but in a different pull request. That's the code we inherited from the old code base we've never had a chance to refactor yet. It's important to not repeat the same mistake in the new code we write. Once it gets it, it's hard to refactor later because of inertia.